node/cmd/guardiand/admintemplate.go

```diff
@@ -1098,9 +1227,9 @@ func parseChainID(name string) (vaa.ChainID, error) {
 	}
 
 	// parse as uint32
-	i, err := strconv.ParseUint(name, 10, 32)
+	i, err := strconv.ParseUint(name, 10, 16)
```

Comment:
Should that comment be updated to `// parse as uint16` now that you changed it? This does make so much more sense too as chains are uint16s. Good change.